creator/ingest_runs/tasks.py

```diff
@@ -85,25 +66,21 @@ def ingest_genomic_workflow_output_manifests(ingest_run):
         f"{list(ingest_run.versions.all())}"
     )
     import time
+
     t_end = time.time() + 10
-    progress = ''
+    progress = ""
     while time.time() < t_end:
         time.sleep(1)
-        progress += '.'
+        progress += "."
         print(progress)
     print(f"Finished ingest run {ingest_run.id}")
 
 
-def get_ingest_run(ingest_run_id):
+def get_ingest_run(ingest_run_uuid):
```

Comment:
No need to convert the uuid to a str anymore since both the mutations already do that before they enqueue the tasks. And now I think you can just replace calls to this method with the one line thats in this method: ingest_run = `IngestRun.objects.get(pk=ingest_run_uuid)`